third_party/zuho/README.amp

```diff
@@ -16,3 +16,5 @@ Local Modifications:
 - Removed String.raw calls (breaks the build)
 - Swapped X-axis projection (original code was horizontally mirroring the source image)
 - Added setImageOrientation(heading, pitch, roll) method to correct the orientation of the texture (credits: Nicolas Barradeau - https://github.com/nicoptere).
+- Added init function for setting up resources.
+- Added properties defined on init to the constructor.
```

Comment:
Typically we also commit a diff file that contains local changes so it's easier to merge upstream updates.